src/main/java/core/mate/academy/model/Truck.java

```diff
@@ -5,6 +5,7 @@
  * Do not remove no-field constructor
  */
 public class Truck extends Machine {
+    int width;
```

Comment:
According to the checklist, you should never want to expose the object fields directly. They should be accessed through special methods (getters and setters). So, please consider changing the access modifier for the field `width` to `private` and provide a getter and a setter for it.